### ISSUE TITLE
should this project provide allowed globals, or rules about them?

### ISSUE THREAD

**Issue by nebulousdog** [lo1tuma/eslint-plugin-mocha#178]
instead of providing globals in `.eslintrc.yml`, should this project provide globals or rules about globals?

For example, I assume many of us have something like the following in our `.eslintrc.yml` file:

```
globals:
  expect: true

  after: true
  afterEach: true
  before: true
  beforeEach: true

  context: true
  describe: true
  it: true

  chai: true
  sinon: true
```

**Comment by lo1tuma** [comment]
Actually I hope that defining those globals is not a common thing people do.
ESLint already provides some predifined environments which declare its globals, see [the official documentations](https://eslint.org/docs/user-guide/configuring#specifying-environments).

So the example config you provided could be replaced with:

```yaml
env:
  mocha: true
```

**Comment by nebulousdog** [comment]
oh that's right! I had that because of my old teaspoon environment and now I can drop it!
